snuba/query/conditions.py

```diff
@@ -86,7 +86,15 @@ def __set_condition_pattern(
         String(operator),
         (
             Param("lhs", lhs),
-            Param("tuple", FunctionCallPattern(String("tuple"), None)),
+            Param(
```

Comment:
It seems like the only place this is tested is the rate limiter but it's in a place for generic use. Can we have a test in `tests/query/test_conditions.py` to test this as a unit?